### PR TITLE
Pr/upgrade xtrans

### PR DESCRIPTION
**Pull request by uli42**
Lift xtrans to current head.

### CODE REVIEW

File: nx-X11/lib/xtrans/Xtrans.c [hunk 1]
```diff
@@ -1230,7 +1230,7 @@ TRANS(MakeAllCLTSServerListeners) (char *port, int *partial, int *count_ret,
  * may be used by it.
  */
 
-#if (defined(SYSV) && defined(i386) && !defined(__SCO__)) || defined(WIN32) || defined(__sxg__) || defined(__UNIXOS2__)
+#if (defined(SYSV) && defined(__i386__) && !defined(__SCO__)) || defined(WIN32) || defined(__sxg__) || defined(__UNIXOS2__)
```
Comment (sunweaver) [inline nx-X11/lib/xtrans/Xtrans.c:1275]: @uli42: please add Backported-to-NX-by: for 79104451e3a0bf77ac239341d68e39162f7bb975.

File: nx-X11/lib/xtrans/Xtransint.h [hunk 2]
```diff
@@ -103,59 +103,6 @@ from The Open Group.
...
-#  else /* !__GNU__ */
-#   if OPEN_MAX > 256
-#    define TRANS_OPEN_MAX 256
-#   else
-#    define TRANS_OPEN_MAX OPEN_MAX
-#   endif
-#  endif /*__GNU__*/
-
-# endif /* TRANS_OPEN_MAX */
-
```
Comment (sunweaver) [inline nx-X11/lib/xtrans/Xtransint.h:169]: @uli42: please add Backported-to-NX-by: for f8cfe723e45834209d70ab0390f3fd782d6872fa.

**Review by sunweaver — CHANGES_REQUESTED** [review]
I left two comments plus this general request: In your commit messages, please don't reference X.org as "upstream", but as "X.org" or "X.org upstream". Thanks.

And another general question: Shouldn't we drop AIX, OS2 and other archs completely? I understand that this PR only tackles Xtrans, but we should actually let another clean-up PR follow-up that removes the old archs completely. Right? I will merge this PR nonetheless and add an issue for each arch that we want to remove.

The rest looks good. Well done!

File: nx-X11/lib/xtrans/Xtransint.h (hunk 2 above)
Comment (uli42) [reply nx-X11/lib/xtrans/Xtransint.h:169]: done

**Comment by uli42** [comment]
I have done the requested changes (+ whitespace fixes). And I agree on removal of these archs. Please open issues for that.

**Review by sunweaver — APPROVED** [review]

**Comment by sunweaver** [comment]
Rebased and merged as 107e72b038bc76953dc97d4ea655c9bde10e5865. Thus, closing.

**Pull request closed without merging**